store/postgres/src/relational_queries.rs

```diff
@@ -1075,6 +1079,21 @@ impl<'a> QueryFilter<'a> {
         Ok(())
     }
 
+    fn filter_block_gte(
+        &self,
+        block_number_gte: &BlockNumber,
+        mut out: AstPass<Pg>,
+    ) -> QueryResult<()> {
+        // constructs a filter for lower(c.block_range) >= $block_number_gte
+        // to allow consumers to filter by a specific change block
+        out.push_sql("lower(c.");
+        out.push_identifier("block_range")?;
```

Comment:
You don't necessarily have to bother with `push_identifier` here since we know that `block_range` is SQL safe; but it would be nice to use the existing constant for that name and make it `out.push_sql(BLOCK_RANGE_COLUMN)`

Reply:
oops, missed that constant, fixed!